starter-code/index.html

```diff
@@ -1,42 +1,106 @@
 <!DOCTYPE html>
+
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:100,300,400,500,600,700" rel="stylesheet">
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+
+    <header>
+
+      <img class="logo" src="./images/spotify-logo.png" alt="logo">
+      <nav>
+        <ul>
+          <li><a href="#0">Premium</a></li>
+          <li><a href="#0">Discover</a></li>
+          <li><a href="#0">Help</a></li>
+          <li><a href="#0">Download</a></li>
+        </ul>
+      </nav>
+
+    </header>
+
+  <section id="landing">
+
+    <h1>Music for everyone.</h1>
+    <p>Spotify is now free on mobile, tablet and computer.
+    <br>Listen to the right music, wherever you are.</p>
+    
+  </section>
+
+  <section >
+
+    <h2><span>What’s on Spotify?</span></h2>
+
+    <div id="articles">
+      <article>
+
+        <img src="./images/music-icon.png" alt="music">
+        <h3>Millions of Songs</h3>
+        <p>There are millions of
+        <br>songs on Spotify</p>
```

Comment:
<br> tags also have a specific semantic meaning, they should only be used when the line break does not serve styling purpose but is part of language conventions, like in poems of addresses.
you can either put : 
```
        <img src="./images/music-icon.png" alt="music">
        <h3>Millions of Songs</h3>
        <p>There are millions of</p>
        <p>songs on Spotify</p>
```
or 
```
        <img src="./images/music-icon.png" alt="music">
        <h3>Millions of Songs</h3>
        <p>There are millions of
        <span>songs on Spotify</span></p>
```
and add a dispaly: block to the span in the css.